docs/userguide/icu_data/buildtool.md

```diff
@@ -204,11 +204,48 @@ summarizes the ICU data files and their corresponding features and categories:
 | Units | `"unit_tree"` | unit/\*.txt | **1.7 MiB** |
 | **OTHER** | `"cnvalias"` <br/> `"misc"` <br/> `"locales_tree"` | mappings/convrtrs.txt <br/> misc/dayPeriods.txt <br/> misc/genderList.txt <br/> misc/numberingSystems.txt <br/> misc/supplementalData.txt <br/> locales/\*.txt | 63 KiB <br/> 19 KiB <br/> 0.5 KiB <br/> 5.6 KiB <br/> 228 KiB <br/> **2.4 MiB** |
 
+#### Additive and Subtractive Modes
+
+The ICU Data Build Tool allows two strategies for selecting features:
+*additive* mode and *subtractive* mode.
+
+The default is to use subtractive mode. This means that all ICU data is
+included, and your configurations can remove or change data from that baseline.
+Additive mode means that you start with an *empty* ICU data file, and you must
+explicitly add the data required for your application.
+
+There are two concrete differences between additive and subtractive mode:
+
+|                         | Additive    | Subtractive |
+|-------------------------|-------------|-------------|
+| Default Feature Filter  | `"exclude"` | `"include"` |
+| Default Resource Filter | `"-/*"`     | `"+/*"`     |
+
+To enable additive mode, add the following key to your filter file:
```

Comment:
Optional: "key" doesn't seem quite right. value/setting/option/directive...?

Reply:
Amended with "setting" instead of "key"